index.html

```diff
@@ -0,0 +1,161 @@
+            <div>
+              <button>html</button>
+              <button>css</button>
+              <button>javascript</button>
+            </div><br>
+            <button class="btn-proj">See Project</button>
+         </div>
+        </div>
+         
+          <!--container 4-->
+          <div class="container3">
+            <div class="img-container">
+              <img class="portolio_img" src="./images/forth_img.png" alt="portfolio image">
+            </div>
+            <div class="work-description">
+            <p class="tittle">Tonic</p>
+            <p class="words"><b class="canopy">CANOPY</b><img class="img" src="./images/Counter.png" alt="count">Back End Dev<img class="img" src="./images/Counter.png" alt="count">2015</p>  
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+              <button>html</button>
+              <button>css</button>
+              <button>javascript</button>
+            </div><br>
+            <button class="btn-proj">See Project</button>
+           </div>
+        </div>
+        
+        <!--ABOUT SECTION-->
+        <div class="about">
+            <h2 class="header">About <br> Myself</h2>
+            <p class="section">
+               I’m a software developer! I can help you build a product,feature or 
+               website Look through some of my work and experience! If you like what 
+               you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+           <p class="footer">LET CONNECT</p>
+             <ul class="imgs">
+              <li><a href="#"><img src="./images/twitter.svg" alt="twitter handle"></a></li>
+              <li><a href="#"><img src="./images/github.jpg" alt="github handle"></a></li>
+              <li><a href="#"><img src="./images/medium.jpg" alt="medium handle"></a></li>
+              <li><a href="#"><img src="./images/angellist.jpg" alt="angellisthandle"></a></li>
+             </ul>
+              <div>
+                <button class="btn-proj">Get my resume</button>
+              </div>
+              <br>
+               <h5 class="lang">Languages</h5>
+                <div class="ellipse">
+                  <button><img src="./images/Ellipse1.png" alt="ellipse"><span>JavaScript</span></button>
+                  <br>
+                  <button><img src="./images/Ellipse2.png" alt="ellipse"><span>HTML</span></button>
+                  <br>
+                   <button><img src="./images/Ellipse3.png" alt="ellipse"><span>CSS</span></button>
+                </div>
+                <div class="parag">
+                  <h4>Frameworks</h4>
+                  <hr>
+                  <p>Skills</p>
+                  <hr>
+                </div>
+                <div class="contact--form">
+                  <form action="https://formspere.io/f/myyoyyer" method="post">
```

Comment:
- [ ] You have done an excellent job with your contact form section :smiley: but when I try to submit the form it throws me an error and I think the problem is because you are not using the [formspree service](https://formspree.io/html) correctly. Please, kindly address this issue with the contact form :smiley: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/145222391-e6659cbc-d9ab-47be-add4-6a2cb4af9e19.png)